overrides/ios-override.json

```diff
@@ -105,6 +105,85 @@
                         "signInRequiredSelector": "[href*=\"//support.google.com/youtube/answer/3037019\"]"
                     }
                 }
+            },
+            "settings": {
```

Comment:
I'm guessing this big difference is because they've diverged now somehow :(. It's fine but also something I really want to fix.